plugins/auto-sizes/tests/e2e/playwright.config.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import * as os from 'os';
+import { fileURLToPath } from 'url';
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * WordPress dependencies
+ */
+import baseConfig from '@wordpress/scripts/config/playwright.config.js';
+
+const config = defineConfig( {
```

Comment:
Why do we need a custom config over what's provided by wp-scripts? IMO the default one should suffice for our needs.